google/cloud/aiplatform/pipeline_jobs.py

```diff
@@ -223,15 +224,25 @@ def __init__(
         if enable_caching is not None:
             _set_enable_caching_value(pipeline_job["pipelineSpec"], enable_caching)
 
-        self._gca_resource = gca_pipeline_job.PipelineJob(
-            display_name=display_name,
-            pipeline_spec=pipeline_job["pipelineSpec"],
-            labels=labels,
-            runtime_config=runtime_config,
-            encryption_spec=initializer.global_config.get_encryption_spec(
+        gca_pipeline_job_compat = gca_pipeline_job
+        pipeline_job_args = {
+            "display_name": display_name,
+            "pipeline_spec": pipeline_job["pipelineSpec"],
+            "labels": labels,
+            "encryption_spec": initializer.global_config.get_encryption_spec(
                 encryption_spec_key_name=encryption_spec_key_name
             ),
-        )
+        }
+
+        if _VALID_AR_URL.match(template_path):
+            gca_pipeline_job_compat = gca_pipeline_job_v1beta1
+            pipeline_job_args["template_uri"] = template_path
```

Comment:
Could you update the docstring for `template_path` in the constructor to include info on AR uris?

Reply:
Done.